lib/pi_piper/sysfs/version.rb

```diff
@@ -0,0 +1,6 @@
+module PiPiper
+  class Driver; end
```

Comment:
I think if you put `require 'pi_piper'` in this file: `lib/pi_piper/sysfs.rb`.
Then it should work because that file will always be loaded before the files within `lib/sysfs/*`